.github/workflows/build-wheels-python-dependent.yml

```diff
@@ -60,16 +60,24 @@ jobs:
       - name: Setup Python
         # GitHub action for MacOS M1 does not have Python <= 3.10
         # Skip setting python on ARM because of missing compatibility: https://github.com/actions/setup-python/issues/108
-        if: (matrix.os != 'macos-latest-xlarge' && matrix.python-version != '3.12') && matrix.os != 'linux-armv7-self-hosted' && matrix.os != 'linux-arm64-self-hosted'
+        if: matrix.os != 'macos-latest-xlarge' && matrix.os != 'linux-armv7-self-hosted' && matrix.os != 'linux-arm64-self-hosted'
         uses: actions/setup-python@v5
         with:
             python-version: ${{ matrix.python-version }}
 
       - name: Setup Python - MacOS M1
         # Temporary solution until Python version for build will be >= 3.10 (GitHub action support)
-        if: matrix.os == 'macos-latest-xlarge' && matrix.python-version != '3.12'
+        if: matrix.os == 'macos-latest-xlarge'
         run: |
           brew install python@${{ matrix.python-version }}
+          # correcting pip installation for later usage
```

Comment:
@jakub-kocka Thank you for the fix. Can you please explain the reason in these comments? 

```
# correcting pip installation for later usage
# Add the required lines to the pip config file
```

We can see this so these comments are redundant. It is more important to comment why is this necessary. This also helps for the feature to see if the given workaround can be removed because it is not necessary anymore.

Reply:
Thank you @dobairoland, you are right.
I have edited the comments to explain the purpose better.